src/Symfony/Component/Messenger/EventListener/StopWorkerOnTimeLimitListener.php

```diff
@@ -30,6 +31,10 @@ public function __construct(int $timeLimitInSeconds, LoggerInterface $logger = n
     {
         $this->timeLimitInSeconds = $timeLimitInSeconds;
         $this->logger = $logger;
+
+        if ($timeLimitInSeconds <= 0) {
```

Comment:
```suggestion
        if (0 >= $timeLimitInSeconds) {
```

To be consistent with the 2 other if statements in your PR

Reply:
I preferred to be consistent with https://github.com/symfony/symfony/blob/ff56fe80de77f4019d28cbe105ce1973b09b580a/src/Symfony/Component/Messenger/EventListener/StopWorkerOnMessageLimitListener.php#L34

(Like for https://github.com/symfony/symfony/pull/48085/files#r1012276670)